src/web3py/extensions/csm.py

```diff
@@ -46,10 +46,8 @@ def get_csm_last_processing_ref_slot(self, blockstamp: BlockStamp) -> SlotNumber
     def get_csm_tree_root(self, blockstamp: BlockStamp) -> HexBytes:
         return self.fee_distributor.tree_root(blockstamp.block_hash)
 
-    def get_csm_tree_cid(self, blockstamp: BlockStamp) -> CID | None:
+    def get_csm_tree_cid(self, blockstamp: BlockStamp) -> CID:
         result = self.fee_distributor.tree_cid(blockstamp.block_hash)
-        if not result:
-            return None
```

Comment:
Why so? Empty string is not a valid CID and will result in CIDv1("") which is not correct.

Reply:
changed